clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -6183,6 +6183,13 @@ void Clang::ConstructJob(Compilation &C, const JobAction &JA,
     }
     if (Args.hasArg(options::OPT_fsycl_unnamed_lambda))
       CmdArgs.push_back("-fsycl-unnamed-lambda");
+
+    // Enable generation of USM address spaces as opt-in.
+    // __ENABLE_USM_ADDR_SPACE__ will be used during compilation of SYCL headers
+    if ((getToolChain().getTriple().getSubArch() ==
+             llvm::Triple::SPIRSubArch_fpga &&
+         Args.hasArg(options::OPT_fsycl_enable_usm_address_spaces)))
```

Comment:
```suggestion
    if (getToolChain().getTriple().getSubArch() ==
            llvm::Triple::SPIRSubArch_fpga &&
        Args.hasArg(options::OPT_fsycl_enable_usm_address_spaces))
```

Reply:
Oops, I incidentally amended and forced-push. I applied this suggestion.